docs/how_to/ssh_agent_forwarding.md

```diff
@@ -0,0 +1,31 @@
+# How to Forward ssh(1) Agent
+
+`ssh(1)` agent forwarding can is activated in the `CONFIG` section of the hosts
+file:
+
+~~~yaml
+HOSTS:
+  ...
+CONFIG:
+  forward_ssh_agent: true
+~~~
+
+Beaker will then be able to use the user's agent on the host machine.  There is
```

Comment:
Please clarify 'host machine' in this line. I'm assuming you mean a "System Under Test (SUT)" (a host in the testing environment), but we use that term or the "beaker coordinator" for the host that beaker is running on.